tools/accuracy_checker/accuracy_checker/launcher/input_feeder.py

```diff
@@ -122,7 +124,7 @@ def fill_non_constant_inputs(self, data_representation_batch):
                 identifiers = data_representation.identifier
                 data = data_representation.data
                 if not isinstance(identifiers, list) and input_regex is None:
-                    input_data = data
+                    input_data = data if num_of_non_constant_inputs <= 1 else data[input_layer_num]
```

Comment:
@liyong4x  could you please describe the reason of these changes? It breaks general pipeline, so we need to find another solution